src/components/Board.vue

```diff
@@ -124,18 +124,15 @@ export default Vue.extend({
 			// info.color = rightClickXorClearing ? '' : this.color;
 			// const oldColor = this.getColor(info.row, info.col);
 		},
-
 		clearPixel(e: MouseEvent) {
-			e.preventDefault();
 			if (this.readonly) {
 				return;
 			}
+			e.preventDefault();
 			const { offsetX: x, offsetY: y } = e;
-			const fillMethod = this.ctx.clearRect;
 			this.ctx.fillStyle = '';
-			fillMethod.apply(this.ctx, [x - x % this.pixelSize, y - y % this.pixelSize, this.pixelSize, this.pixelSize]);
+			this.ctx.clearRect.apply(this.ctx, [x - x % this.pixelSize, y - y % this.pixelSize, this.pixelSize, this.pixelSize]);
```

Comment:
```suggestion
			this.ctx.clearRect([x - x % this.pixelSize, y - y % this.pixelSize, this.pixelSize, this.pixelSize]);
```
The `this` binding was only required when the function was being stored in a variable